server/routes/induction/common/additionalTrainingController.ts

```diff
@@ -1,15 +1,14 @@
 import { NextFunction, Request, RequestHandler, Response } from 'express'
 import type { InductionDto } from 'inductionDto'
 import type { AdditionalTrainingForm } from 'inductionForms'
-import InductionController from './inductionController'
 import AdditionalTrainingView from './additionalTrainingView'
 import AdditionalTrainingValue from '../../../enums/additionalTrainingValue'
 import { asArray } from '../../../utils/utils'
 
 /**
  * Abstract controller class defining functionality common to both the Create and Update Induction journeys.
  */
-export default abstract class AdditionalTrainingController extends InductionController {
```

Comment:
`InductionController` was the superclass for all Induction related controllers. It contained all the functions required for manipulating `pageFlowHistory` in order to tell if Check Your Answers was the previous page or not.
Now that we've migrated away from that approach, all of the controllers no longer need to extend `InductionController`; and the class itself can be deleted (see below)